### DISCUSSION TITLE
Terminal output logging

### DISCUSSION THREAD

**Discussion by tomoniizuma** [wezterm/wezterm#2862] [Q&A] [answered]
Is there any way to write the output from the terminal to a log file with wezterm ?

**Comment by wez** [comment, marked as answer]
`wezterm record` will record an [asciinema](https://asciinema.org/) compatible asciicast. `wezterm replay` will play one back.

**Comment by wez** [comment]
https://wezfurlong.org/wezterm/config/lua/pane/get_logical_lines_as_text.html and https://wezfurlong.org/wezterm/config/lua/pane/get_lines_as_text.html can be used to capture lines from the scrollback. 
[This example from the docs shows it in use](https://wezfurlong.org/wezterm/config/lua/wezterm/on.html#custom-events)
